tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,6 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert item not in vendor.inventory
+    assert len(vendor.inventory) == 3
+    assert result == False
```

Comment:
A more "pythonic" style to write this is `assert not result` like you did in the wave 6 tests line 240, though this works just the same!